convert2rhel/backup/files.py

```diff
@@ -65,7 +65,7 @@ def enable(self):
                 logger.critical_no_exit("Error({}): {}".format(err.errno, err.strerror))
                 raise exceptions.CriticalError(
                     id_="FAILED_TO_SAVE_FILE_TO_BACKUP_DIR",
-                    title="Failed to copy file to the backup directory.",
+                    title="Failed to copy file to the backup directory",
```

Comment:
To be more like "Failed to copy a certificate" code change i saw in `backup/certs` I think it makes sense to do it like that here too
```suggestion
                    title="Failed to copy a file to the backup directory",
```